qtdialogs.py

```diff
@@ -5673,7 +5673,7 @@ def __init__(self, pytx, wlt, parent, main, mode=None, \
          pytx = ustx.getPyTxSignedIfPossible()
 
 
-      self.pytx = pytx.copy()
+      self.pytx = pytx.copyWithoutWitness()
```

Comment:
I don't quite remember, but I think it had to do with hashing the tx. This was probably changed before I did the other stuff to correct tx hashing.